client/src/assets/components/profile/EditProfile.jsx

```diff
@@ -136,6 +162,13 @@ const EditProfile = () => {
           headers: { Authorization: `Bearer ${auth?.accessToken}` },
           withCredentials: true
         })
+      if (response?.status === 200) {
+        showSuccess(`${field} updated successfully`);
+        setErrMsg('');
+      }
+      else {
+        showError(`${field} update failed`);
+      }
```

Comment:
The catch error block below this also catches the error and sends the response so this isn't needed.

The catch (err) is overwriting the response causing the error toast not to show.